components/blitz/src/ome/services/blitz/repo/ManagedImportRequestI.java

```diff
@@ -387,30 +390,36 @@ private void detectKnownAnnotations() throws Exception {
             if (NSAUTOCLOSE.value.equals(ns)) {
                 autoClose = true;
             } else if (NSTARGETTEMPLATE.value.equals(ns)) {
-                ome.model.annotations.CommentAnnotation ca =
-                        (ome.model.annotations.CommentAnnotation) ann;
-                if (settings.userSpecifiedTarget != null) {
-                    // TODO: Exception
-                    String kls = settings.userSpecifiedTarget.getClass().getSimpleName();
-                    long id = settings.userSpecifiedTarget.getId().getValue();
-                    log.error("User-specified template target '{}' AND {}:{}",
-                            ca.getTextValue(), kls, id);
-                    continue;
+                ome.model.annotations.CommentAnnotation
+                    ca = (ome.model.annotations.CommentAnnotation) ann;
+                if (ca.getId().longValue() > maxCAId) {
+                    maxCAId = ca.getId().longValue();
+                    lastCA = ca;
                 }
-
+            }
+        }
+        if (lastCA != null) {
+            if (settings.userSpecifiedTarget != null) {
+                // TODO: Exception
+                String kls = settings.userSpecifiedTarget.getClass().getSimpleName();
+                long id = settings.userSpecifiedTarget.getId().getValue();
+                log.error("User-specified template target '{}' AND {}:{}",
```

Comment:
You tell me! I just preserved the original line from https://github.com/openmicroscopy/openmicroscopy/blame/develop/components/blitz/src/ome/services/blitz/repo/ManagedImportRequestI.java#L396 which I think is yours.

Would you rather it was set at INFO? I can add a commit here if so.